### PR TITLE
Added a just-ask command to prompt a user to just ask the question

### PR DESCRIPTION
**Pull request by daredevil82**
Command to prompt the a user to just ask a question, rather than asking permission or if anyone is around for a question.

Would like some feedback on the text of the prompt

### CODE REVIEW

File: sirbot-pyslackers/endpoints/slack/commands.py [hunk 1]
```diff
@@ -9,16 +9,32 @@
...
     plugin.on_command("/report", report)
     plugin.on_command("/save", save_conversation)
 
 
-async def ask(command, app):
+async def just_ask(command, app):
+    slack = app.plugins["slack"]
+    response = Message()
+    response["channel"] = command["channel_id"]
+    response["unfurl_links"] = False
```
Comment (ovv) [inline sirbot-pyslackers/endpoints/slack/commands.py]: This is not necessary. By default it's at `True` but in this case there is no link in the message, so i can be left off.

File: sirbot-pyslackers/endpoints/slack/commands.py [hunk 2]
```diff
@@ -9,16 +9,32 @@
...
+
+    response["text"] = (
+        "If you have a question, please just ask it. Do not ask to ask; do not ask for topic experts; "
+        "do not DM or ping random users. We cannot begin to answer a question until we actually get a "
+        "question. Please skip the formalities and just ask."
+    )
```
Comment (ovv) [inline sirbot-pyslackers/endpoints/slack/commands.py]: Seems good to me but might be worth asking in #community and/or #admin for feedback

File: sirbot-pyslackers/endpoints/slack/commands.py (hunk 1 above)
Comment (daredevil82) [reply sirbot-pyslackers/endpoints/slack/commands.py]: good catch.

File: sirbot-pyslackers/endpoints/slack/commands.py (hunk 2 above)
Comment (daredevil82) [reply sirbot-pyslackers/endpoints/slack/commands.py]: already did in #community_projects, but will do in community as well.

File: sirbot-pyslackers/endpoints/slack/commands.py (hunk 1 above)
Comment (daredevil82) [reply sirbot-pyslackers/endpoints/slack/commands.py]: I left that in due to adding a link as shown by wmont  in #community_projects

File: sirbot-pyslackers/endpoints/slack/commands.py (hunk 2 above)
Comment (daredevil82) [reply sirbot-pyslackers/endpoints/slack/commands.py]: Changed this to Jabob's suggestion

**Pull request merged**